lib/jekyll/watcher.rb

```diff
@@ -18,12 +18,13 @@ module Watcher
     # Returns nothing.
     def watch(options, site = nil)
       ENV["LISTEN_GEM_DEBUGGING"] ||= "1" if options['verbose']
+      @source = options['source']
 
       site ||= Jekyll::Site.new(options)
       listener = build_listener(site, options)
       listener.start
 
-      Jekyll.logger.info "Auto-regeneration:", "enabled for '#{options["source"]}'"
+      Jekyll.logger.info "Auto-regeneration:", "enabled for #{@source}".green
```

Comment:
Thanks! Instance variables should only be used when the value isn't per-method call, e.g. a Listen configuration file. The way to do what you're pointing out here is to make a class instead of a module and to instantiate it with the right variables then call some method that would begin the watching process. That'd have to be a new class though, as we can't change Watcher from a module to a class. It's cleaner to just pass the options or pass the source to each function here as it's needed.